comp/core/workloadmeta/collectors/internal/docker/docker.go

```diff
@@ -297,6 +297,11 @@ func (c *collector) buildCollectorEvent(ctx context.Context, ev *docker.Containe
 			}
 		}
 
+		restartPolicy := "no"
```

Comment:
it can be good to define the different restartPolicy value as a `const string` 
or even try to do a kind of golang enum

something like:
```golang
type RestartPolicyValueType string
const (
    RestartPolicyAlways RestartPolicyValueType = "always"
    RestartPolicyNo RestartPolicyValueType = "no"
)
```